{% extends "base.html" %}

{% load static %}

{% block head_title %}{{ block.super }} | Events{% endblock %}
{% block page_title %}Events{% endblock %}

{% block content %}
<div>
  <h2><span style="color:red;">CANCELLED</span>: Building minimal cells: From non-essential genes identification to genome engineering<span class="details">, March 27-29, 2018 @ Bordeaux, FR</span>
    <a class="details" href="http://www.crg.eu/en/event/coursescrg-building-minimal-cells-non-essential-genes-identification-genome-engineering-0"><i class="fi-link"></i></a>
  </h2>
  
  <p><a href="{% get_static_prefix %}/doc/poster_building_minimal_cells_course_2018.pdf"><img src="{% get_static_prefix %}/img/poster_building_minimal_cells_course_2018.png" style="width:474px; float:right; margin-left:20px; padding:2px; border: 1px dotted #0186a8;"></a></a>Minimal cells are extremely significant scientific objects, as their study will bring us closer to understanding what life really is. In addition to this fundamental question, minimal cells are also a keystone of the development of synthetic biology and biotechnology. Indeed, the ability to remove non-essential genes from a genome is crucial to the production of a streamlined organism. Building these minimal cells, perfectly tailored for a given task, promises significant improvement in a wide array bioprocesses ranging from metabolite production to the creation of synthetic cells. The goal of the MiniCell Workshop on “Building Minimal Cells” is to provide young investigators cutting-edge training in the design and production of genome-reduced micro-organisms, for both fundamental and applied sciences.</p>
  
  <h3>Why participate?</h3>
  <p>The course will revolve around two axes: the identification of essential and non-essential genes in a given micro-organism; and the design, validation and implementation of genome engineering tools to produce genome-reduced cells. Participants will be trained in the production and sequencing of transposon libraries in bacteria, Tn-seq data analysis, and the in vitro rapid prototyping and testing of CRISPR-Cas9 genome edition tools. The course will feature a complete day of seminar, with didactic presentations by scientists at the forefront of their fields. This seminar will introduce the participants to the core concepts underlying the production of genome-reduced cells, in the model bacterium Bacillus subtilis and Mycoplasma genitalium. The next two days will be dedicated to hands-on tutorials and experiments in both wet and in silico biology.</p>
  
  <h3>Who is the course for?</h3>
  <p>The course is designed for PhD students and postdoctoral scholars who wish to gain training in genome engineering and genome minimization. Students should already have a good foundation in molecular biology.</p>
  
  <h3>Pre-requisites</h3>
  <p>The course will focus on teaching students theory and techniques for essential genes identification and genome engineering. Participation in the course requires prior knowledge of basic molecular biology, in particular the manipulation of nucleic acids (DNA &amp; RNA), as well as a basic knowledge of the principles of high-throughput sequencing.</p>
  
  <h3>Organizers</h3>
  <p>
  <b>Yonathan Arfi</b>, Bordeaux University, Bordeaux, France<br/>
  <b>Alain Blanchard</b>, Bordeaux University, Bordeaux, France<br/>
  <b>Damjana Kastelic</b>, CRG, Barcelona, Spain<br/>
  <b>Carole Lartigue</b>, INRA, Bordeaux, France<br/>
  <b>Maria Lluch-Senar</b>, CRG, Barcelona, Spain<br/>
  <b>Luis Serrano-Pubul</b>, CRG, Barcelona, Spain<br/>
  <b>Pascal Sirand-Pugnet</b>, Bordeaux University, Bordeaux, France
  </p>
 
  <h3>Instructors and speakers</h3>
  <p>
  <b>Mickey Gjerris</b>, Copehaguen University, Denmark<br/>
  <b>John Glass</b>, JCVI, San Diego, USA<br/>
  <b>Jonathan Karr</b>, Icahn School of Medicine at Mount Sinai, New York, USA<br/>
  <b>Jochen Hecht</b>, CRG, Barcelona, Spain<br/>
  <b>Carole Lartigue</b>, INRA, Bordeaux, France<br/>
  <b>Samuel Miravet</b>, CRG, Barcelona, Spain<br/>
  <b>Pascal Sirand-Pugnet</b>, Bordeaux University, Bordeaux, France<br/>
  <b>Jörg Stülke</b>, Georg-August-Universität, Göttingen, Germany
  </p>
  
  <h3>Schedule</h3>
  <p>Duration of the course: The course will start on Tuesday March 27th at 9 am and finish on Thursday March 29th at 5pm.</p>

  <h3>Number of participants</h3>
  <p>20</p>
 
  <h3>Location</h3>
  <p>ISVV –Institute of Vine &amp; Wine Science <a href="https://goo.gl/maps/2E2Ad9Us4SC2"><i class="fi-link"></i></a><br/>
  210 Chemin de Leysotte, 33140 Villenave-d'Ornon, France</p>
  
  <h3>Registration fee</h3>
  <p>250€ (+ 21% VAT) for academic researchers and 500€ (+21% VAT) for industry.</p>
  
  <h3>How to apply</h3> 
  <p>The application is available at the course website <a href="http://www.crg.eu/en/event/coursescrg-building-minimal-cells-non-essential-genes-identification-genome-engineering-0"><i class="fi-link"></i></a>. The application deadline is March 1, 2018.</p>
  
  <p>Participants will be selected according to their CV, motivation letter and description of their work. Participants will be notified if they have been selected for the course by the beginning of March, 2018.</p>
   
  <h3>More information</h3>
  <p>Please visit the course website <a href="http://www.crg.eu/en/event/coursescrg-building-minimal-cells-non-essential-genes-identification-genome-engineering-0"><i class="fi-link"></i></a> or contact <a href="mailto:training@crg.eu">training@crg.eu</a>.</p>
</div>

<div>
  <h2>Whole-cell modeling summer school<span class="details">, September 4-7, 2017 @ CRG, Barcelona ES</span>
    <a class="details" href="http://www.wholecell.org/school-2016.html"><i class="fi-link"></i></a>
  </h2>
  
  <p>Whole-cell models are promising tools for predicting phenotype from genotype by accounting for every individual gene and cell function. Whole-cell modeling has the potential to enable rational bioengineering and precision medicine. However, significant work remains to develop fully complete and accurate whole-cell models. The goal of the 2017 Whole-Cell Modeling Summer School is to provide young investigators cutting-edge training in large-scale dynamical modeling and model integration.</p>

  <p>The course will be the first course focused on multi-algorithm whole-cell modeling. It will teach strategies for building and managing large models which aren't covered by any other course including multi-algorithm modeling, model organism database curation, surrogate modeling, and software development. The five-day course will feature didactic lectures, interactive hands-on tutorials, and student research talks. The mornings will feature lectures on modeling individual pathways. The afternoons will feature interactive hands-on tutorials on building and analyzing multi-algorithm models to generate and evaluate hypotheses. Throughout the course, students will work toward building a small whole-cell model. In addition, the course will include student talks to enable students to share their own research.</p>
  
  <h3>Schedule</h3>
  <p>The course will be four days long. The first day will feature an introductory lecture and ice-breaker activites. Day 2-4 will feature a combination of lectures, hands-on tutorials, and group discussions.</p>
  
  <h3>How to apply</h3> 
  <p>The course application is available online <a href="http://www.crg.eu/en/event/coursecrg-whole-cell-modeling"><i class="fi-link"></i></a>. The application asks for brief descriptions of why you want to participate in the course, your background, and your current research, as well as your CV.</p>
   
  <h3>More information</h3>
  <p>Please visit the <a href="http://www.wholecell.org/school-2017">course website</a> for more information.</p>
</div>

<div>
  <h2>Whole-cell modeling summer school<span class="details">, April 3-8, 2016 @ CRG, Barcelona ES</span>
    <a class="details" href="http://www.wholecell.org/school-2016"><i class="fi-link"></i></a>
  </h2>
  
  <p>The summer school will teach students how to build whole-cell models through didactic lectures and hands-on active learning exercises on biological pathways, dynamical modeling, hybrid modeling, and strategies for managing model complexity. Students will learn about key intracellular pathways, methods for modeling biological pathways, and the latest methods for constructing models which represent multiple pathways.</p>
  
  <h3>Schedule</h3>
  <p>The course will be five days long. Each morning will include 2 lectures that will introduce students to biological concepts in metabolism, cellular signaling, and cell cycle regulation, as well as computational concepts in logical, constraint-based, ordinary differential equation, and hybrid modeling. Each afternoon will feature 2 tutorials which will provide students hands-on instruction in model building, model composition, and simulation. The course will also provide students opportunities to present their research, as well as network with the other students and faculty.</p>
  
  <h3>How to apply</h3> 
  <p>The summer school will be open to graduate students and postdoctoral scholars. An online application will be available on the <a href="http://www.wholecell.org/school-2016">course website</a> in Fall 2015.</p>
   
  <h3>More information</h3>
  <p>Please visit the <a href="http://www.wholecell.org/school-2016">course website</a> for more information.</p>
</div>

{% endblock %}